submissions/alyonafuria/html-css_popup/index.html

```diff
@@ -0,0 +1,180 @@
+<!doctype html>
+
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <title>Hooli popup</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <nav>
+        <ul class="navigation">
+            <li><a href="#" class="upperButton">Mail</a></li>
+            <li><a href="#" class="upperButton">Images</a></li>
+            <li>
+                <input type="checkbox" id="toggle" />
+                <label for="toggle" class="popup-button"><img src="assets/popup-icons/popup-button.png" alt="appGrid"></label>
+                <div class="overlay">
+                    <div class="popup">
+                        <a href="#" class="el">
+                            <img src="assets/popup-icons/calendar.png" alt="Calendar" width="40px" height="40px">
+                            <figcaption>Calendar</figcaption>
+                        </a>
+                        <a href="#" class="el">
+                            <img src="assets/popup-icons/mail.png" alt="Mail" width="60px" height="40px">
+                            <figcaption>Mail</figcaption>
+                        </a>
```

Comment:
It's block code and the same blocks wrap in `li`.

Reply:
sorry, I don't understand

Reply:
Wrap your block `class="overlay"` in `ul` or `ol` tag and after it wrap all your links `a`,  inside block `overlay`, in tag `li`

Reply:
oh, thanks!